src/defaults.sh

```diff
@@ -33,8 +33,10 @@
 [ -z "$IP_BINARY" ] && IP_BINARY=$(command -v ip)
 [ -z "$IPTABLES" ] && IPTABLES=iptables_wrapper
 [ -z "$IPTABLES_BINARY" ] && IPTABLES_BINARY=$(command -v iptables)
+[ -z "$IPTABLES_BINARY" ] && IPTABLES_BINARY=$(command -v iptables-nft)
```

Comment:
Please switch the order, so that iptables-nft is evaluated first. 
Otherwise the first line, with iptables, populates the IPTABLES_BINARY variable, and the added second line is worthless. 
Or did I understood the the logic wrong?

(There will likely be iptables present for the near future, due to scrambled dependencies...)


The same comment for the ip6tables below.

Reply:
Hmm, I'm not sure that is the right thing to do in general, though? Having both binaries available is really a weird setup, and I'm not sure we can assume any order in general.

Maybe a better solution for openwrt would be to explicitly set IP{,6}TABLES_BINARY in /etc/sqm/sqm.conf for the platform?